docs/cli.md

```diff
@@ -24,6 +24,68 @@ Welcome to the SB CLI!
 
 The following lists `sb` commands usages and examples:
 
+### `sb benchmark list`
+
+List benchmarks which match the name.
```

Comment:
If I know the name, why do we list it? What's the goal for this feature? I think you would like to list all benchmarks.

Reply:
I think you support regex, but you don't mention here.